README.md

```diff
@@ -1,10 +1,14 @@
-# MyBusinessAPI :money_mouth_face:
+# MyBarberAPI :money_mouth_face:
 
 ## Descripción del problema :spiral_notepad:
 
 Dado un caso real, en donde a una barbería normalmente acudía la gente sin previo aviso para esperar a poder ser atendida, tuvo que adaptarse a las nuevas normativas y habilitar un teléfono para la reserva de citas debido al COVID-19.
 
-Se plantea a partir de esta situación crear una API para la gestión de negocios. En esta se almacenarán datos del negocio gestionado, de los productos que pueda ofrecer para venta online, de los clientes, las citas de los mismos, de los pedidos que realicen, etc.
+El problema aparece a la hora de reserva de los distintos tipos de cita, cada uno con diferente duración. En este proceso una mala disposición de las diferentes citas puede suponer un número de menor de citas diarias, disminuyendo así las ganancias del negocio.
+
+Como consecuencia de la reserva de citas pueden aparecer problemas referenciados a modificaciones, anulaciones o no asistencia a citas. Esto afecta directamente al desarrollo del trabajo en el negocio.
+
+Además otro de los problemas que aparecen es la cantidad de articulos en el pedido mensual de productos para el negocio. Estos productos son utilizados para las citas y dependiendo del tipo de cita se agotarán antes o después.
```

Comment:
OK, como problema parece que tiene un potencial de generar una lógica de negocio adecuada, aunque me habría gustado que especificaras un poco más is vas a predecir algo. No queda muy claro en las citas qué es lo que vas a hacer con ellas.